packages/backend/src/modules/auth/config/auth.config.spec.ts

```
@@ -0,0 +1,90 @@
+
+      // Re-import to get new config
+      jest.resetModules();
+      const { defaultAuthConfig: newConfig } = require('./auth.config');
+
+      expect(newConfig.jwt.secret).toBe('test-secret');
+
+      // Cleanup
+      if (originalEnv) {
+        process.env.JWT_SECRET = originalEnv;
+      } else {
+        delete process.env.JWT_SECRET;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)